setup.py

```diff
@@ -10,14 +10,14 @@
 
 # NOTE: a typical webserver is included with framework dependencies if necessary
 trigger_extras = {"PyYAML>=5.1", "lxml>=4.3.1", "mock==3.*"}
-django_extras = {"Django"} | trigger_extras
-falcon_extras = {"falcon", "gunicorn", "uwsgi"} | trigger_extras
-flask_extras = {"Flask"} | trigger_extras
-pyramid_extras = {"pyramid", "waitress"} | trigger_extras
+django_extras = {"Django==3.*"} | trigger_extras
+falcon_extras = {"falcon==2.*", "gunicorn==20.1", "uwsgi==2.0.*"} | trigger_extras
+flask_extras = {"Flask==1.*"} | trigger_extras
+pyramid_extras = {"pyramid==1.*", "waitress==2.0.*"} | trigger_extras
 wsgi_extras = trigger_extras
-bottle_extras = {"bottle"} | trigger_extras
+bottle_extras = {"bottle==0.*"} | trigger_extras
 
-dev_extras = {"WebTest", "gunicorn", "tox"}
+dev_extras = {"WebTest==2.0.*", "gunicorn==20.1", "tox==3.*"}
```

Comment:
we don't actually need gunicorn under `dev_extras` anymore, could you remove it?

Reply:
sure!